README.md

```diff
@@ -181,6 +181,23 @@ Class names like `Options` and `Level` might be too common, if you get a conflic
     })
     ...
 
+#### Angular Universal
+
+To use it with Angular Universal add angular2-logger to App.module normaly and in node.module provide these custom options.   
+
+```javascript
+import { Options } from "angular2-logger/core";
+
+@NgModule({
+  bootstrap: [ AppComponent ],
+  imports: [ AppModule, ... ],
+  providers: [
+    { provide: Options, useValue: { global: false, store: false } }, // stop angular2-logger from defining global variables.
```

Comment:
That line is not a decoration, its there for a reason, its part of the functionality, having different configurations won't bring the code back. Consider that the configuration file its just the default or initial configuration, this configuration can change and it needs to be persisted in the storage so the user doesn't have to be changing it back every time he refreshes the page.